www/src/components/navigation.js

```diff
@@ -128,15 +128,6 @@ const Navigation = ({ pathname }) => {
             <NavItem linkTo="/blog/">Blog</NavItem>
             <NavItem linkTo="/showcase/">Showcase</NavItem>
             <NavItem linkTo="/contributing/">Contributing</NavItem>
-            {/* <li css={styles.li}>
```

Comment:
You want to put these other changes into another PR?

Reply:
It's more difficult to review multiple changes in one PR

Reply:
raising a separate PR